core/src/main/java/org/primefaces/extensions/component/clockpicker/ClockPicker.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (c) 2011-2024 PrimeFaces Extensions
+ *
+ *  Permission is hereby granted, free of charge, to any person obtaining a copy
+ *  of this software and associated documentation files (the "Software"), to deal
+ *  in the Software without restriction, including without limitation the rights
+ *  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ *  copies of the Software, and to permit persons to whom the Software is
+ *  furnished to do so, subject to the following conditions:
+ *
+ *  The above copyright notice and this permission notice shall be included in
+ *  all copies or substantial portions of the Software.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ *  THE SOFTWARE.
+ */
+package org.primefaces.extensions.component.clockpicker;
+
+import java.util.Locale;
+
+import javax.faces.application.ResourceDependency;
+import javax.faces.component.html.HtmlInputText;
+import javax.faces.context.FacesContext;
+
+import org.primefaces.component.api.Widget;
+import org.primefaces.util.Constants;
+import org.primefaces.util.LocaleUtils;
+
+@ResourceDependency(library = "primefaces", name = "components.css")
+@ResourceDependency(library = "primefaces", name = "jquery/jquery.js")
+@ResourceDependency(library = "primefaces", name = "jquery/jquery-plugins.js")
+@ResourceDependency(library = "primefaces", name = "core.js")
+@ResourceDependency(library = "primefaces-extensions", name = "clockpicker/0-clockpicker.css")
+@ResourceDependency(library = "primefaces-extensions", name = "clockpicker/0-clockpicker.js")
+@ResourceDependency(library = "primefaces-extensions", name = "clockpicker/1-clockpicker-widget.js")
```

Comment:
Please aggregate the scripts in the pom.xml. See for example https://github.com/primefaces-extensions/primefaces-extensions/blob/ad5024fb95812bb51a55ee82fa9b1c6a2c615d50/core/pom.xml#L194-L206

Reply:
Agreed if these are not minified you can just combine themwith `clockpicker/**`